openquake/server/db/actions.py

```diff
@@ -283,8 +283,7 @@ def get_outputs(db, job_id):
     'loss_maps-stats': 'Asset Loss Maps Statistics',
     'agg_curves-rlzs': 'Aggregate Loss Curves',
     'agg_curves-stats': 'Aggregate Loss Curves Statistics',
-    'agg_losses-rlzs': 'Aggregate Losses',
-    'agg_losses-stats': 'Aggregate Losses Statistics',
+    'aggrisk': 'Aggregate Losses',
```

Comment:
What's the difference between `aggrisk` on this line and `agg_risk` in the line below?

Reply:
The `agg_risk` below is used in volcanic risk. It would be nice to unify things (if possible) but it is left for the future.